src/Resources/views/Layout/Footer/Grid/_customer_care.html.twig

```diff
@@ -4,7 +4,15 @@
         <h4 class="ui inverted header">{{ 'sylius.ui.customer_care'|trans  }}</h4>
         <div class="ui inverted link list">
             {% for item in footer_customer_care %}
-                <a class="item">{{ item.label }}</a>
+                {% set noreferrer = item.noreferrer %}
+                {% set noopener = item.noopener %}
+                {% set nofollow = item.nofollow %}
+                {% set hasRel = noreferrer or noopener or nofollow %}
+                <a href="{{ item.url }}" class="item" {% if item.targetBlank %}target="_blank"{% endif %}
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Do you think it's smart to create a specific template for items and manage the logic inside ?

People will be able to customize it in their themes

Reply:
@maximehuran I think it could be nice but I also think that for most sites it would be unnecessary as depending where the menus is displayed it must dramatically change in the html.

It's also kinda made in this template with the macro: `src/Resources/views/Layout/Header/_menu.html.twig`